src/Jackett/Indexers/nCore.cs

```diff
@@ -0,0 +1,214 @@
+        {
+            List<string> searchurls = new List<string>();
+            var searchString = query.SanitizedSearchTerm + " " + query.GetEpisodeSearchString();
+            searchurls.Add(String.Format(SearchUrl, 1, HttpUtility.UrlEncode(searchString)));
+            searchurls = getSearchUrls(searchurls, searchString).Result;
+
+            List<ReleaseInfo> releases = new List<ReleaseInfo>();
+            foreach (string url in searchurls)
+            {
+                var results = await client.GetStringAsync(url);
+                try
+                {
+                    CQ dom = results;
+
+                    ReleaseInfo release;
+                    var rows = dom[".box_torrent_all"].Find(".box_torrent");
+
+                    foreach (var row in rows)
+                    {
+                        CQ qRow = row.Cq();
+
+                        release = new ReleaseInfo();
+                        var torrentTxt = qRow.Find(".torrent_txt").Find("a").Get(0);
+                        if (torrentTxt == null) continue;
+                        release.Title = torrentTxt.GetAttribute("title");
+                        release.Description = release.Title;
+                        release.MinimumRatio = 1;
+                        release.MinimumSeedTime = 172800;
+
+                        string downloadLink = SiteLink + torrentTxt.GetAttribute("href");
+                        string downloadId = downloadLink.Substring(downloadLink.IndexOf("&id=") + 4);
+
+                        release.Link = new Uri(SiteLink.ToString() + "torrents.php?action=download&id=" + downloadId);
+                        release.Comments = new Uri(SiteLink.ToString() + "torrents.php?action=details&id=" + downloadId);
+                        release.Guid = new Uri(release.Comments.ToString() + "#comments"); ;
+                        release.Seeders = ParseUtil.CoerceInt(qRow.Find(".box_s2").Find("a").First().Text());
+                        release.Peers = ParseUtil.CoerceInt(qRow.Find(".box_l2").Find("a").First().Text()) + release.Seeders;
+                        release.PublishDate = DateTime.Parse(qRow.Find(".box_feltoltve2").Get(0).InnerHTML.Replace("<br />", " "), CultureInfo.InvariantCulture);
+                        string[] sizeSplit = qRow.Find(".box_meret2").Get(0).InnerText.Split(' ');
+                        release.Size = ReleaseInfo.GetBytes(sizeSplit[1].ToLower(), ParseUtil.CoerceFloat(sizeSplit[0]));
+
+                        releases.Add(release);
+                    }
+                }
+                catch (Exception ex)
+                {
+                    OnParseError(results, ex);
+                }
+            }
+
+            return releases.ToArray();
+        }
+
+        private async Task<List<string>> getSearchUrls(List<string> searchurls, string searchString)
+        {
+            var results = await client.GetStringAsync(searchurls.First());
```

Comment:
Could we not get the first page then get the links of that so we don't hit the server twice?